cleanlab/experimental/label_issues_batched.py

```diff
@@ -20,15 +20,18 @@
 You can also use this to estimate label quality scores or the number of label issues
 for big datasets with limited memory.
 
-The recommended usage is two passes over your data: 
+With default settings, the results returned from this method closely approximate those returned from:
```

Comment:
Unsure which method this is referring to since it is in the introduction, maybe clarify "with default settings, the results returned from `find_label_issues_batched()` or example script..."

Reply:
this method = Implementation of find_label_issues that does not need much memory by operating in mini-batches.

They are all the same method. "This method" here was referring to mathematical procedure, not a code method. To avoid confusion I've switched it to be called "This approach"

Reply:
Sounds good! It was the same line as in the actual `find_label_issues_batched()` function docstring so felt a bit strange.